usecases/traverser/near_params_vector.go

```diff
@@ -343,20 +345,32 @@ func (v *nearParamsVector) crossClassFindVector(ctx context.Context, id strfmt.U
 			if len(res[0].Vectors) == 0 || res[0].Vectors[targetVector] == nil {
 				return nil, "", fmt.Errorf("vector not found for target: %v", targetVector)
 			}
-		} else {
-			if len(res[0].Vectors) == 1 {
-				for key, vec := range res[0].Vectors {
-					v, ok := vec.([]float32)
-					if !ok {
-						return nil, "", fmt.Errorf("unrecognized vector type: %T", vec)
-					}
-					return v, key, nil
+			vec, ok := res[0].Vectors[targetVector].([]float32)
+			if !ok {
+				return nil, "", fmt.Errorf("unrecognized vector type: %T", vec)
+			}
+			return vec, targetVector, nil
+		}
+
+		if len(res[0].Vector) > 0 {
+			return res[0].Vector, "", nil
+		}
+
+		if len(res[0].Vectors) == 0 {
+			return nil, "", nil
+		}
```

Comment:
can we also put those check after `if len(res[0].Vectors) == 1`? same reason: want to prioritize the named vectors configuration